File: app/src/main/java/com/example/shiranpeer/burritolocator/screen/MainActivityMvpView.java
package com.example.shiranpeer.burritolocator.screen;

import com.example.shiranpeer.burritolocator.model.Place;
import com.example.shiranpeer.burritolocator.screen.base.MvpView;

import java.util.List;

public interface MainActivityMvpView extends MvpView {
     void showNearbyPlaces(List<Place> places, String nextPageToken);

     void showNearbyAdditionalPlaces(List<Place> places, String nextPageToken);

     void hideProgressBar();

     void showProgressBar();
}

File: app/src/main/java/com/example/shiranpeer/burritolocator/screen/mainActivity/MainActivityMvpView.kt
package com.example.shiranpeer.burritolocator.screen.mainActivity

import com.example.shiranpeer.burritolocator.model.Place
import com.example.shiranpeer.burritolocator.screen.base.MvpView

interface MainActivityMvpView : MvpView {
    fun showNearbyPlaces(places: List<Place>, nextPageToken: String)

    fun showNearbyAdditionalPlaces(places: List<Place>, nextPageToken: String)

    fun hideProgressBar()

    fun showProgressBar()

    fun showGPSErrorMessage()

    fun showNetworkErrorMessageForLoading()

    fun hideProgressBarLoader()
}
